onboarding/web-infra.md

```diff
@@ -0,0 +1,23 @@
+# Web Infrastructure Onboarding
+
+This document is an outline of the tasks necessary to onboard a new member of the Web Infrastructure team.
+
+Please note these tasks should only be performed after the person's nomination has passed.
+
+## Tasks
+
+Tasks prefixed by `Member:` are for the new member to complete.
+Tasks prefixed by `Web Admin:` are for a member of `@nodejs/web-admins` to complete.
```

Comment:
I feel like we can just say "A web admin should ...", "The onboardee should ...". in each task